internal/cli/atlas/quickstart/prompt.go

```diff
@@ -142,3 +152,73 @@ func newProfileDocQuestionOpenBrowser() *survey.Confirm {
 		Message: "Do you want more information to set up your profile [This will open www.mongodb.com on your browser]?",
 	}
 }
+
+func providerQuestion(provider string) *survey.Question {
+	if provider == "" {
+		return newClusterProviderQuestion()
+	}
+
+	return nil
+}
+
+func clusterNameQuestion(clusterName string) *survey.Question {
+	if clusterName == "" {
+		message := ""
+		clusterName = autogeneratedName()
+		if clusterName != "" {
+			message = fmt.Sprintf(" [Press Enter to use the auto-generated name '%s']", clusterName)
+		}
+		return newClusterNameQuestion(clusterName, message)
+	}
+
+	return nil
```

Comment:
[nit] prefer the shortest code inside the `if` also let's avoid modifying the input
```suggestion
	if clusterName != "" {
		return nil
	}
	message := ""
	newClusterName := autogeneratedName()
	if newClusterName != "" {
		message = fmt.Sprintf(" [Press Enter to use the auto-generated name '%s']", newClusterName)
	}
	return newClusterNameQuestion(newClusterName, message)
```

let's apply the shortest `if` approach to the other methods here